graphiti_core/search/search_utils.py

```
@@ -261,13 +270,16 @@ async def edge_similarity_search(
         routing_='r',
     )
 
+    if driver.provider == 'falkordb':
+        records = [dict(zip(header, row)) for row in records]
```

Ruff (0.11.9)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)